config/initializers/high_voltage.rb

```diff
@@ -1,3 +1,4 @@
 HighVoltage.configure do |config|
   config.home_page = 'homepage'
+  config.route_drawer = HighVoltage::RouteDrawers::Root
```

Comment:
With this line, the High Voltage gem will draw routes like `/foo` instead of `/pages/foo`.